routers/api/v1/repo/issue.go

```diff
@@ -105,7 +110,8 @@ func SearchIssues(ctx *context.APIContext) {
 		issueIDs, err = issue_indexer.SearchIssuesByKeyword(repoIDs, keyword)
 	}
 
-	if splitted := strings.Split(ctx.Query("labels"), ","); len(splitted) > 0 {
+	splitted := strings.Split(ctx.Query("labels"), ",")
+	if len(splitted) > 0 {
```

Comment:
[`len(splitted)` is always >= 1](https://play.golang.org/p/booAmz8TOtM). For example: `strings.Split("", ",") == []string{""}`, which has a `len()` of 1. You need to check whether `labels == ""` or `splitted[0] == ""` is empty too.

Reply:
What I mean is, you need to check `routers/api/v1/repo/issue.go:108` because the `if()` condition is wrong.

Reply:
Gotcha, now I get it (facepalm). I fixed the condition